relay-sampling/src/lib.rs

```diff
@@ -957,14 +1002,14 @@ mod tests {
 
     #[test]
     fn test_field_value_provider_trace_filled() {
-        let tc = TraceContext {
+        let tc = DynamicSamplingContext {
```

Comment:
nit:
```suggestion
        let dynsc = DynamicSamplingContext {
```
or similar like `dsc`, `dync`...

Reply:
renamed to dsc